meson.build

```diff
@@ -1674,6 +1674,7 @@ if get_option('cplayer')
     install_data('etc/_mpv.zsh', install_dir: zsh_install_dir, rename: '_mpv')
 
     install_data('etc/mpv.desktop', install_dir: join_paths(datadir, 'applications'))
+    install_data('etc/mpv.metainfo.xml', install_dir: join_paths(datadir, 'metainfo'))
```

Comment:
shouldn't that be `io.mpv.mpv.metainfo.xml` according to the spec?

Reply:
It doesn't matter. But if you want to rename it, desktop file must be renamed too.